src/status_im2/contexts/quo_preview/list_items/community_list.cljs

```diff
@@ -2,39 +2,33 @@
   (:require [quo2.core :as quo]
```

Comment:
FYI: The diff in this file is just to show in this PR how the abstraction is used. Nothing of importance changed.